### ISSUE TITLE
Optimizition: use lm.fit instead of lm

### ISSUE THREAD

**Issue by durszlaczek** [MI2DataLab/factorMerger#60]
lm.fit is faster than lm. However, some statistics must be calculated separately (e.g. group means).

**Comment by pbiecek** [comment]
premature optimization is the root of all evil

**Comment by durszlaczek** [comment]
Agreed, just put it here not to forget about it.  That's why there is no assignee for this issue.
